integration-tests/mysql_test.go

```diff
@@ -170,35 +161,34 @@ var _ = Describe("Mysql", func() {
 
 		BeforeEach(func() {
 			var err error
-			instanceID, err = broker.Provision("csb-google-mysql", customMySQLPlan["name"].(string), nil)
+			instanceID, err = broker.Provision(mySQLServiceName, customMySQLPlanName, map[string]any{"tier": "db-n1-standard-1"})
 
 			Expect(err).NotTo(HaveOccurred())
-			Expect(mockTerraform.FirstTerraformInvocationVars()).To(
```

Comment:
**question**: Why is this no longer the case?

Reply:
Good question @jhvhs! the check is performed multiple times through the test code base (e.g: line 60).
I deleted the `Reset` operation because I want to emphasise that after the provision operation, if we update the instance and the updating fails, we do not perform another Terraform invocation (line 190)